ggml.c

```diff
@@ -729,6 +729,93 @@ static void quantize_row_q4_1(const float * restrict x, void * restrict vy, int
     }
 }
 
+void quantize_row_q4_1(const float * restrict x, void * restrict vy, int k) {
```

Comment:
```suggestion
static void quantize_row_q4_1(const float * restrict x, void * restrict vy, int k) {
```

Reply:
Hmm, I usually can commit these changes from GH web UI, but now it is not possible for some reason